STAR-Delegation/draft-ietf-acme-star-delegation.md

```diff
@@ -362,16 +366,83 @@ Potentially, since it holds the STAR cert private key, it could request the
 revocation of a single STAR certificate.  However, STAR explicitly disables the
 revokeCert interface.
 
-# CDNI Use Cases
+# CSR Template
+
+The CSR template is used to express and constrain the shape of the CSR that the
+NDC uses to request the certificate.  The CSR is used for every CSR created
+under the same delegation.  Its validation is a critical element in the
+security of the whole delegation mechanism.
+
+The CSR template is defined using JSON Schema {{!I-D.handrews-json-schema}}, a
+mature, widely used format, which is a natural fit for the JSON-centric ACME.
+
+Instead of defining every possible CSR attribute, this document takes a
+minimalist approach by declaring only the minimum attribute set and deferring
+the registration of further, more specific, attributes to future documents.
+Critically, this document establishes the necessary IANA registry and
+registration rules (see {{csr-template-registry}}).
+
+## Rules
+
+TODO
+
+## Example
+
+The CSR template in {{fig-csr-template}} represents one possible CSR template
+governing the delegation exchanges provided in the rest of this document.
```

Comment:
I'm confused: this looks like a cross between a template (JSON schema) and a specific instance. I think we need an actual template here, and make it normative, not an "example".

Reply:
Fully agree -- this is what the above section "Rules" is meant for, except it's empty at the moment. The problem is this needs a bit more thought than what I can allocate right now :-(